testlib/src/test/java/io/spine/testing/logging/LoggerTestTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.testing.logging;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@DisplayName("`LoggerTest` should")
+class LoggerTestTest {
+
+    private LoggerTest test;
+    private Level previousLevel;
+    private Level newLevel;
+
+    @BeforeEach
+    void createFixture() {
+        previousLevel = Level.SEVERE;
+        newLevel = Level.FINE;
+        jdkLogger().setLevel(previousLevel);
+        test = new TestFixture(getClass(), newLevel);
+    }
+
+    private Logger jdkLogger() {
+        return Logger.getLogger(getClass().getName());
+    }
+
+    @Test
+    @DisplayName("assign logging class")
+    void loggingClass() {
+        assertThat(test.loggingClass())
+                .isEqualTo(getClass());
```

Comment:
Why do you move `isEqualTo` to the next line?

There is an `isNotNull` call below that is on the same line. Let's use similar formatting everywhere.

Reply:
I've put null check on new line.